src/repositories/donationRepository.ts

```diff
@@ -537,36 +537,16 @@ export async function isVerifiedDonationExistsInQfRound(params: {
   }
 }
 
-export async function findRelevantDonations(
+export async function findDonationsByProjectIdWitchUseDonationBox(
   startDate: Date,
   endDate: Date,
   givethProjectId: number,
```

Comment:
You can rename `givethProjectId` to `projectId`

Reply:
Done